xml/admin_caasp_advanced_config.xml

```diff
@@ -142,10 +142,10 @@ mountSecret: ceph-user1-secret
 </screen>
    <para>
     If you want the Rook &ceph; agent to require a <literal>mountUser</literal>
-    and <literal>mountSecret</literal> to be set in StorageClasses using Rook,
-    you need to set the environment variable
+    and <literal>mountSecret</literal> to be set in StorageClasses using
+    &rook;, you need to set the environment variable
     <varname>AGENT_MOUNT_SECURITY_MODE</varname> to
-    <literal>Restricted</literal> on the Rook &ceph; operator deployment.
+    <literal>Restricted</literal> on the &rook; &ceph; operator deployment.
```

Comment:
Since Rook Operator is the name of the software, shall we use the version where both words are capitalized throughout the whole document?

Reply:
This one is definitely wrong - good catch. There's the occasion where it's Rook Operator (tool) and Rook operator (person)